apps/web/pages/event-types/index.tsx

```diff
@@ -720,10 +720,7 @@ const EventTypeListHeading = ({
       <Avatar
         alt={profile?.name || ""}
         href={teamId ? `/settings/teams/${teamId}/profile` : "/settings/my-account/profile"}
-        imageSrc={
-          `${orgBranding?.fullDomain ?? WEBAPP_URL}${teamId ? "/team" : ""}/${profile.slug}/avatar.png` ||
-          undefined
-        }
+        imageSrc={profile?.image || undefined}
```

Comment:
this isn't working in case of orgs

Reply:
you can create org using http://localhost:3000/settings/organizations/new

Reply:
It is asking to verify email. I dont know how to setup an org